docs/NcObject.md

```diff
@@ -6,7 +6,15 @@ The control class model for NcObject is listed in the [Framework](Framework.md#n
 
 The `role` is a structural identifier which MUST be persisted across restarts.
 
-Object ids (`oid` property) may be constant across system restarts in which case they MUST be signaled using the `constandOid` property by settings its value to `true`.
+The `role` of an object MUST be unique within its containing block.
+
+Object ids (`oid` property) MUST uniquely identity objects in the device model.
+
+Once an object in the device model is allocated an object id, it MUST not change until the device undergoes a reboot.
+
+If an object has a constant oid (has the same value across system restarts) it MUST signal this behaviour using the `constandOid` property by settings its value to `true`.
```

Comment:
Should `constantOid` be instead of `constandOid`?

Reply:
Definitely! Thanks for spotting!